src/utils/cosmosdb_model.ts

```diff
@@ -264,6 +268,19 @@ export abstract class CosmosdbModel<
     );
   }
 
+  /**
+   * Add or update the ttl of the document identified by the searchKey
+   */
+
+  public updateTTL(
```

Comment:
why do we want a special patch operation only for the `ttl` field?

Reply:
I thought it could be useful to have a method like that so you can pass a `NonNegativeNumber` directly, but I agree that this could be redundant